sources/packages/api/src/route-controllers/education-program/models/save-education-program.dto.ts

```diff
@@ -8,9 +8,44 @@ import {
   getISODateOnlyString,
   getUserFullName,
 } from "../../../utilities";
+
 /**
- * Dto that represents education programs form object.
+ * Dto that represents education program with offering form object.
```

Comment:
Comment can be slightly adjusted. It has education program details only. hasOffering is a meta-data of program itself.

Reply:
I have updated the comment.